pyworkflow/utils/utils.py

```
@@ -729,13 +729,20 @@ def getEnvVariable(variableName, default=None, exceptionMsg=None):
 
 
 def pluginNotFound(plugName, errorMsg='', doRaise=False):
```

What happens with the doRaise?
Just now we found it useful to use doRaise in the "mixed test" when we start importing plugins and they are not installed.

---

`doRaise = True`  should raise an error telling the same than the warning in the case that `doRaise = False`. Have you test it??

---

Ouch! Now I see!! It's delete it... My fault... I will review it!

Thanks @pconesa !

---

;-)

---

Fixed! :)